From f85405d733dd4f1cf05879eab3b852ce81eab3ea Mon Sep 17 00:00:00 2001
From: dyep <dyep49@gmail.com>
Date: Thu, 18 Oct 2018 15:08:22 -0700
Subject: [PATCH] OS-7220 Snapshots are not unmounted on VM pre-halt Reviewed
 by: Rob Johnston <rob.johnston@joyent.com> Reviewed by: Mike Gerdts
 <mike.gerdts@joyent.com> Approved by: Mike Gerdts <mike.gerdts@joyent.com>

---
 overlay/generic/usr/lib/brand/jcommon/statechange | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/generic/usr/lib/brand/jcommon/statechange b/overlay/generic/usr/lib/brand/jcommon/statechange
index da3bd811..58eae5ab 100644
--- a/overlay/generic/usr/lib/brand/jcommon/statechange
+++ b/overlay/generic/usr/lib/brand/jcommon/statechange
@@ -876,7 +876,7 @@ pre)
 		setup_fs
 		;;
 	4)	# pre-halt
-		[[ -n "$jst_snowsnap" ]] && cleanup_snapshots
+		[[ -n "$jst_showsnap" ]] && cleanup_snapshots
 		cleanup_net
 		;;
 	esac
-- 
2.21.0

